### PR TITLE
Golang storage rules

### PR DESCRIPTION
**Pull request by ankit-privado**
Added Rules for storages

### CODE REVIEW

File: rules/sinks/storages/dynamodb/go.yaml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+sinks:
+
+  - id: Storages.AmazonDynamoDB.ReadAndWrite
+    name: Amazon Dynamo DB
+    domains:
+      - aws.amazon.com
+      - amazon.com
+    patterns:
+      - "(?i)(github.com/aws/aws-sdk-go/service/dynamodb).*(PutItem|GetItem|UpdateItem|DeleteItem)"
```
Comment (hiteshbedre) [inline rules/sinks/storages/dynamodb/go.yaml:9]: Lets divide this into Read and ReadAndWrite

File: rules/sinks/storages/mongodb/go.yaml [hunk 2]
```diff
@@ -0,0 +1,17 @@
+sinks:
+
+  - id: Storages.MongoDB.Read
+    name: MongoDB(Read)
+    domains:
+      - mongodb.com
+    patterns:
+      - "(?i)(go.mongodb.org/mongo-driver/mongo).*(InsertOne|Find|)"
```
Comment (hiteshbedre) [inline rules/sinks/storages/mongodb/go.yaml:8]: InsertOne can be shifted to Write

Comment (ankit-privado) [reply rules/sinks/storages/mongodb/go.yaml:8]: done.

File: rules/sinks/storages/dynamodb/go.yaml (hunk 1 above)
Comment (ankit-privado) [reply rules/sinks/storages/dynamodb/go.yaml:9]: done.

**Review by hiteshbedre — APPROVED** [review]
LGTM

**Pull request merged**